mono/metadata/exception.c

```diff
@@ -216,13 +200,15 @@ create_exception_two_strings (MonoClass *klass, MonoString *a1, MonoString *a2,
 		break;
 	}
 
-	args [0] = a1;
-	args [1] = a2;
+	gpointer args [ ] = { MONO_HANDLE_RAW (a1), MONO_HANDLE_RAW (a2) };
 
-	mono_runtime_invoke_checked (method, o, args, error);
-	return_val_if_nok (error, NULL);
-
-	return (MonoException *) o;
+	mono_runtime_invoke_handle (method, o, args, error);
+	goto_if_nok (error, return_null);
```

Comment:
(nitpicking)
I think in this case an `if` is clearer:
```c
   if (!is_ok (error)) {
    o = mono_new_null ();
  }
exit:
```

Reply:
I like the consistency but ok.